lam4doc/entrypoints/ui/views.py

```diff
@@ -27,23 +30,33 @@ def index():
     return render_template('index.html', title='LAM index page')
 
 
-@app.route('/lam-report', methods=['GET'])
+@app.route('/lam-report', methods=['GET', 'POST'])
```

Comment:
do we need POST? is that additional functionality or just a mirror of GET? (if the latter then let's reduce to GET alone)

Reply:
the `post` is required as the view handles the form which allows the client to select the type of report to retrieve, if only `get` is specified, the method will not work